pkg/virt-config/deprecation/feature-gates.go

```diff
@@ -53,8 +53,6 @@ const (
 	// CommonInstancetypesDeploymentGate enables the deployment of common-instancetypes by virt-operator
 	CommonInstancetypesDeploymentGate = "CommonInstancetypesDeploymentGate" // GA
 
-	PasstGate   = "Passt"   // Deprecated
```

Comment:
Wording: This package is "deprecation" but we have GA features here. For me this is more of "graduation". Can we do something with this wording or am I the only one getting confused by this?